app/services/actual/import.rb

```diff
@@ -41,7 +41,13 @@ def import_row(row, index)
     end
 
     def add_error(row_number, column, cell_value, message)
-      @errors << Error.new(row_number, Converter::FIELDS[column], cell_value, message)
+      column_string = if column == :values
```

Comment:
Although I'm usually against ternaries for return values in Ruby, I think I find them more readable in variable assignments, but this might just be me! What do you think?

Reply:
Me too, but I wondered whether it was a little complicated/long here:

`column_string = column == :values ? "Actual Value/Refund Value" : Converter::FIELDS[column]`

Maybe it's just the colons and quote marks in the symbol, string and namespacing making it look messy - it's probably fine!

Reply:
I think with parentheses (suggested by Standard) it's fine:

`column_string = (column == :values) ? "Actual Value/Refund Value" : Converter::FIELDS[column]`